py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2782,3 +2782,28 @@ def aten_ops_roll(
         args[1],
         args_bounds_check(args, 2, []),
     )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.index_select.default)
+@enforce_tensor_types(
+    {
+        0: (TRTTensor,),
+        2: (TRTTensor,),
+    }
+)
+def aten_ops_index_select(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.index.index_select(
```

Comment:
It seems that the `index_select` function could be put into `select.py`

Reply:
I moved `index_select` inside `select.py`. Thank you!